api/src/test/java/com/epam/pipeline/manager/pipeline/FolderTemplateManagerTest.java

```diff
@@ -168,5 +117,117 @@ public void createFolderFromTemplateTest() throws IOException {
                 CHILD_TEMPLATE_FOLDER_NAME_1);
         Assert.assertNotNull(savedChildFolder);
         Assert.assertEquals(rootFolderId, savedChildFolder.getParentId());
```

Comment:
All the asserts in this test made sense when `folderManager`, `metadataManager`, `dataStorageManager` and `folderCrudManager` fields weren't mock. Now when they are the test can be simplified significantly.

Could you please get rid of all the mocked call results asserting?

Reply:
Fixed